.gitignore

```diff
@@ -1 +1,3 @@
 .DS_Store
+Gemfile.lock
```

Comment:
I've never totally understood the ethic behind ignoring the lockfile, but I know that's a thing, and I'm willing to go with it. If so, can you rm the file that's tracked for now?